csv/src/main/java/com/fasterxml/jackson/dataformat/csv/impl/CsvDecoder.java

```diff
@@ -65,12 +69,14 @@
     protected boolean _trimSpaces;
 
     protected boolean _allowComments;
-    
+
```

Comment:
Need to add `@since`, once we figure which branch this should go in. Same for new methods.

Reply:
I added `@since 2.10` to the feature  + the new method that replace the skip comments.
I am not sure where else I should put it.

Reply:
I usually add to method and field, but not a big deal for internal fields.